content/en/repository.html

```diff
@@ -14,6 +14,7 @@
 
 <h1 id="let-s-encrypt-subscriber-agreement">Let's Encrypt Subscriber Agreement</h1>
 <ul>
+  <li><a href="/documents/LE-SA-v1.6-August-18-2025.pdf">v1.6, Effective August 18, 2025</a></li>
   <li><a href="/documents/LE-SA-v1.5-February-24-2025.pdf">v1.5, February 24, 2025</a></li>
```

Comment:
This line needs to move down to between lines 24 and 25.

Reply:
That's still the current effective agreement. The new line has a future effective date.